src/script/view_model/bindings/CommonBindings.ts

```diff
@@ -34,23 +34,26 @@ import {viewportObserver} from '../../ui/viewportObserver';
 /**
  * Use it on the drop area.
  */
+type KOEvent<T = Event> = JQuery.Event & {currentTarget: Element; originalEvent: T};
```

Comment:
Let's move the type above the comment since the comment belongs to the function below.